samples/sensor/sngcja5/src/main.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2023 Panasonic Industrial Devices Europe GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/sensor.h>
+
+static struct sensor_value sensor_values_particle_count[6];
+static struct sensor_value sensor_values_pm_1_0;
+static struct sensor_value sensor_values_pm_2_5;
+static struct sensor_value sensor_values_pm_10;
+
+void main(void)
+{
+	int ret;
+	const struct device *const dev = DEVICE_DT_GET_ONE(panasonic_sngcja5);
```

Comment:
Please make this sample more generic so it can support other particle sensor drivers. See `samples/sensor/accel_polling` for example

Reply:
Did that on last pull request update.